src/SayMore/UI/ComponentEditors/ContributorsEditor.cs

```diff
@@ -252,12 +253,16 @@ private void SaveContributors()
 		/// --------------------------------------------------------------------------------------
 		/// Get the participants list from the Sessions contributions
 		/// --------------------------------------------------------------------------------------
-		private string GetParticipants()
+		private string GetParticipants(bool withRoles)
 		{
 			string participants = string.Empty;
+			var names = new HashSet<string>();
 			foreach (Contribution contributor in _model.Contributions)
 			{
-				participants += contributor.ContributorName + " (" + contributor.Role.Name + "); ";
+				if (!withRoles && names.Contains(contributor.ContributorName))
+					continue;
+				names.Add(contributor.ContributorName);
+				participants += contributor.ContributorName + (withRoles? " (" + contributor.Role.Name + "); " : ";");
```

Comment:
Probably doesn't really matter, but the older version of SayMore did have a space following the semi-colon to separate the participants in the list.